package.json

```diff
@@ -1,9 +1,13 @@
 {
```

Comment:
need to install the semantic release packages. See https://github.com/Pocket/terraform-modules/blob/main/package.json#L114-L120